lib/server.js

```diff
@@ -19,6 +19,17 @@ function defaultResolveServerListeningText (address) {
   return `Server listening at ${address}`
 }
 
+function addAbortSignalOption (server, listenOptions) {
+  const onAborted = () => {
+    this.close()
```

Comment:
so basically `const onAborted = this.close` ?

Reply:
Why not

`function addAbortSignalOption (onAborted, server, listenOptions) {`

then you just do in line 62

`addAbortSignalOption(this.close, server, listenOptions) `

Reply:
@Uzlopak Good comment, fixing!